client/devdocs/design/playground.jsx

```diff
@@ -133,18 +145,94 @@ class DesignAssets extends React.Component {
 		}
 	}
 
+	state = {
+		code: `<Main>
+    <HeaderCake actionText="Fun" actionIcon="status">Welcome to the Playground</HeaderCake>
+  	<Button primary onClick={
+  		function() {
+  			alert( 'World' )
+  		}
+  	}>
+  		<Gridicon icon="code" /> Hello
+  	</Button>
+  	<br /><hr /><br />
+  	<ActionCard
+  		headerText={ 'Change the code above' }
+  		mainText={ 'The playground lets you drop in components and play with values. Its experiemental and likely will break.' }
```

Comment:
While you are here, how about fixing this `Its` (should be `It's`)

Reply:
I think this should have been fixed in master already.